packages/modules/viewer-react/src/types.ts

```diff
@@ -134,20 +141,59 @@ export interface ViewerInitializerParams extends ViewerIModelAppOptions {
 
 export interface ConnectedViewerProps {
   iTwinId: string;
-  iModelId: string;
+  iModelId?: string;
   changeSetId?: string;
 }
-
 export interface FileViewerProps {
   /** Path to local snapshot or briefcase */
   filePath: string;
 }
 
-export interface BlankViewerProps {
-  blankConnection: BlankConnectionProps;
+export type BlankViewerProps = {
```

Comment:
wondering if we should split this type further down and have it be a union of the deprecated way and the updated/recommended usage. youre kina already doing this 

as is rn, BlankViewerProps === BlankViewerPropsWithoutDeprecation (ignoring the deprecated jsdoc tag)

Reply:
Also reworked.